src/NServiceBus.Persistence.ServiceFabric/SagaPersister/ServiceFabricSagaAttribute.cs

```diff
@@ -2,10 +2,24 @@
 {
     using System;
 
+    /// <summary>
+    /// Allows to influence the collection name and/or the saga data name when a saga is stored into reliable collections.
+    /// </summary>
     [AttributeUsage(AttributeTargets.Class, Inherited = false)]
     public sealed class ServiceFabricSagaAttribute : Attribute
     {
+        /// <summary>
+        /// The saga data name is primarily used for backwards compatibility. By default the saga data type name is used
+        /// to create a determinstic identifier to store the saga into the reliable collection. Renaming a saga data
+        /// will therefore influence the identifier generation. By setting the saga data name to saga data name before
```

Comment:
> By setting the saga data name to saga data name before the renaming the identifier generation will rename stable.

Perhaps this?

By setting the saga data name to saga data **_type_** name prior to ~~before the~~ renaming the identifier generation will rename stable.

Also, I don't follow the "will rename stable." part. The only way to keep the collection name stable is to _not_ use a saga data type and instead assign an arbitrary value.